std_msgs/Header header
uint8 id
UserInfo[] user_infos
